tests/unit/services/gmail/test_unread.py

```
@@ -0,0 +1,305 @@
+
+        scan_unread_by_sender(limit=100, inbox_only=False)
+
+        # Check that the query does not include "in:inbox"
+        call_args = mock_service.users.return_value.messages.return_value.list.call_args
+        query = call_args.kwargs.get("q", "")
+        assert "is:unread" in query
+        assert "in:inbox" not in query
+
+    @patch("app.services.gmail.unread.get_gmail_service")
+    @patch("app.services.gmail.unread.time.sleep")
+    def test_successful_scan_groups_by_sender(self, mock_sleep, mock_get_service):
```

Ruff (0.14.10)

Unused method argument: `mock_sleep`

(ARG002)